config/element-api.php

```diff
@@ -586,36 +585,6 @@ function getListing($locale)
     ];
 }
 
-/**
- * API Endpoint: Get flexible content
- * Get a page using the flexible content field model
- */
-function getFlexibleContent($locale)
```

Comment:
ironically the least flexible method in the API since it was hard-limited to one section...